sdk/openai/openai/review/openai-types.api.md

```diff
@@ -100,18 +74,14 @@ export interface AzureSearchChatExtensionConfiguration extends AzureChatExtensio
 
 // @public
 export interface AzureSearchChatExtensionParameters {
-    allow_partial_result?: boolean;
     authentication?: OnYourDataAuthenticationOptions;
```

Comment:
Would you happen to know why there's a difference between TypeSpec and the spec file here? In the `inference.json` file, all `authenticaction` properties for Azure Chat Extensions is required for Azure Search Param, for example [here]( https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/AzureOpenAI/inference/stable/2024-06-01/inference.json#L1696-L1700), but in TypeSpec, it's optional [here](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/OpenAI.Inference/models/completions/extensions/oyd/azure_cosmosdb_options.tsp#L47)

Reply:
One other difference is the type of authentication allowed. In `inference.json` file, the `authentication` property does not have Connection String options [here](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/data-plane/AzureOpenAI/inference/stable/2024-06-01/inference.json#L1704-L1713), but it is allowed in TypeSpec

Reply:
connection strings auth is not GA.

Reply:
Good catch for authentication being required, it has to be required.